sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.PartitionProcessor.cs

```diff
@@ -0,0 +1,356 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Processor;
+using Microsoft.Azure.WebJobs.EventHubs.Processor;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Azure.Core.Diagnostics;
+
+namespace Microsoft.Azure.WebJobs.EventHubs.Listeners
+{
+    internal sealed partial class EventHubListener
+	{
```

Comment:
nit:  Indenting seems to be off across the class.

Reply:
had to redo all the tabs in this file but I think I fixed it